tests/unit/test_git_url_watcher.py

```diff
@@ -29,6 +29,17 @@
     run_command,
 )
 
+RETRYING_PARAMETERS: dict[str, Any] = {
+    "stop": stop_after_attempt(10),
+    "wait": wait_fixed(3),
+}
+
+
+def sleep_1_sec_to_make_commit_timestamp_unique():
+    # git seems to keep track of commit datetimes only up to seconds, so we need to sleep here to prevent both commits
+    # having the same timestamp (FIXME)
```

Comment:
maybe refrain from writing FIXME in comments. they tend to be picked up by linters.

Reply:
done thx